common/AppRoutes.tsx

```diff
@@ -24,6 +24,7 @@ const layoutConfig = (path: string, isMobile: boolean): LayoutConfig => {
         fullW: true,
         bgColor: '#fff'
       };
+    case ROUTE_PATHS.SETTINGS.path:
```

Comment:
There is the same case below (`case ROUTE_PATHS.SETTINGS.path:`) which is unreachable after adding this line.
Link to the case: https://github.com/MyCryptoHQ/MyCrypto/pull/3184/files#diff-58901591c0a8f6591a7b3c4d5b1c1904R33

Reply:
😱 Good catch!